lib/engines/content_object_store/app/models/content_object_store/content_block_version.rb

```diff
@@ -0,0 +1,5 @@
+class ContentObjectStore::ContentBlockVersion < ApplicationRecord
+  belongs_to :item, polymorphic: true
+  validates :event, presence: true
```

Comment:
Could `event` be an enum?

Reply:
that's probably a good idea! instead of `create` we would have to use `created` though, because otherwise it conflicts with the ActiveRecord `create` method, but that would actually benefit us as it's how we will render/read it in the UI...